tests/use_case/__init__.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from tests.fixtures.environment_variables import *
+
+os.environ['TABLE'] = TEST_DATASET
+os.environ["DATABASE"] = PDB_TEST
+os.environ["POSTGRES_USERNAME"] = POSTGRES
+os.environ["POSTGRES_PASSWORD"] = POSTGRES
+os.environ["POSTGRES_HOST"] = LOCALHOST
+os.environ["POSTGRES_PORT"] = PORT
```

Comment:
Same comment on parameters in may different files

Reply:
It is so that they can be run independently.